test/extended/security/supplemental_groups.go

```diff
@@ -33,7 +36,18 @@ var _ = g.Describe("[security] supplemental groups", func() {
 			fsGroup := int64(1111)
 			supGroup := int64(2222)
 
-			_, err := oc.AsAdmin().Run("adm").Args("policy", "add-scc-to-user", "anyuid", oc.Username()).Output()
+			err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+				_, err := oc.AsAdmin().Run("adm").Args("policy", "add-scc-to-user", "anyuid", oc.Username()).Output()
+				if err == nil {
```

Comment:
Can we do this stuff as a type switch?

Reply:
I don't think I know how to make it more concise with `switch`.

Reply:
I was thinking a type switch, but it in this case you do not need that since you only care about one type.  You can remove the `nil` check though:

```go
err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
	_, err := oc.AsAdmin().Run("adm").Args("policy", "add-scc-to-user", "anyuid", oc.Username()).Output()
	if exitErr, ok := err.(*exutil.ExitError); ok {
		if strings.HasPrefix(exitErr.StdErr, "Error from server (Conflict):") {
			// the retry.RetryOnConflict expects "conflict" error, let's provide it with one
			return errors.NewConflict(schema.GroupResource{}, "", err)
		}
	}
	return err
})
o.Expect(err).NotTo(o.HaveOccurred())
```